docs/sources/alerting/alerting-rules/alerting-migration/migration-api.md

```diff
@@ -0,0 +1,92 @@
+---
+description: Use the Grafana Alerting API import tool to convert your datasource managed alert rules into Grafana managed alert rules
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Import data source-managed alert rules with Grafana Mimirtool
+menuTitle: API alert rules import
+weight: 601
+refs:
+---
+
+# Import data source-managed alert rules with Grafana Mimirtool
+
+You can convert data source-managed alert rules to Grafana-managed alert rules with the Grafana import tool in the Grafana user interface, or you can convert them with the Grafana Mimirtool command-line tool. This guide tells you how to use Mimirtool to import your data source-managed alert rules.
+
+## Before you begin
+
+To import data source-managed alert rules with Grafana Mimirtool, you need to have the Grafana Mimirtool command-line tool installed.
+
+You need a service account the following [RBAC permissions](/docs/grafana/latest/administration/roles-and-permissions/access-control/):
+
+- Alerting: Rules Reader
+- Alerting: Rules Writer
+- Alerting: Set provisioning status
+- Datasources: Reader
+- Folders: Creator
+- Folders: Reader
+- Folders: Writer
+
+You also need to create a service account token with your service account. Refer to the [documentation for more information on service accounts and service account tokens](/docs/grafana/latest/administration/service-accounts/)
+
+## How it works
+
+When data source-managed alert rules are converted to Grafana-managed alert rules, the following are applied to the Grafana-managed alert rules:
+
+- All rules are given `rule_query_offset` offset value of 1m.
+- The `missing_series_evals_to_resolve` is set to 1 for the new rules.
+- The newly created rules are given unique UIDs.
```

Comment:
People also can specify the UID using `__grafana_alert_rule_uid__` label. If this label is set on the alert rule, we use its value as the UID instead of generating the UID ourselves. Can we also add something to the docs about this please?